frontend/src/Screen3.js

```diff
@@ -102,18 +104,20 @@ const MyForm = () => (
 
 export const Screen3 = () => (
   <CenterContent>
-    {/* <Breadcrumb>
+    <Breadcrumb>
       <Link to={'/'}>
         <Trans>Landing Page</Trans>
       </Link>
       <Link to={'/form1'}>
-        <Trans>What happened?</Trans>
+        <Trans>What Happened</Trans>
       </Link>
-      <Link to={'/form2'}>
-        <Trans>What was involved?</Trans>
+      <Link to="{'/form2'}">
```

Comment:
those `"` are messing up the link. TBH, we probably don't need the curly braces, ie `<Link to="/form2">` works I think.